simulation/sub8_simulation/sub8_sim_tools/physics/vehicle.py

```diff
@@ -5,10 +5,13 @@
 from sub8_ros_tools import rosmsg_to_numpy, normalize
 from sub8_sim_tools.physics.physics import Box
 from sub8_simulation.srv import SimSetPose, SimSetPoseResponse
+from sub8_simulation.msg import DVLSim
```

Comment:
Is the file for DVLSim commited?

Also - put it in sub8_msgs and make it [like this](https://github.com/uf-mil/software-common/blob/master/uf_common/msg/VelocityMeasurements.msg), so we can use the old UKF with minimal adjustment